internal/controller/controllers/controller_event_wrapper.go

```diff
@@ -56,16 +56,64 @@ func (c *controllerEventsWrapper) GetEvents(clusterID strfmt.UUID, hostID *strfm
 
 func (c *controllerEventsWrapper) SendClusterEvent(ctx context.Context, event events.ClusterEvent) {
 	c.events.SendClusterEvent(ctx, event)
+
+	cluster, err := common.GetClusterFromDB(c.db, *event.GetClusterId(), common.SkipEagerLoading)
```

Comment:
consider adding a helper function, this code is duplicated.

Reply:
done